nix/nixpkgs.nix

```diff
@@ -0,0 +1,13 @@
+# Pinned nixpkgs version.
+
+let
```

Comment:
@nicholasbishop The dedicated `nix/nixpkgs.nix` file is rather a convention than a requirement. Could also be included inline in `shell.nix`. However, this way it is a little cleaner, I think.